### DISCUSSION TITLE
The modal with the options to personalize mercadopago cookies is not showing when clicking on "Configurar Cookies"

### DISCUSSION THREAD

**Discussion by caco13** [mercadopago/sdk-js#102] [General]
I'm testing the integration with Checkout Pro, following the instructions in Checkout Pro docs for the interface. I get the payment button "Pagar" and getting the mercadopro modal showing correctly.

But when I click in "Configurar cookies" I get the new modal empty, without the the instructions for personalize the cookies.

Using the sample code for the template page:

```   
    <script src="https://sdk.mercadopago.com/js/v2"></script>

    <script>
        const mp = new MercadoPago('{{ public_key }}', {
            locale: 'pt-BR'
        });

        mp.checkout({
            preference: {
                id: '{{ preference.id }}'
            },
            render: {
                container: '.cho-container',
                label: 'Pagar',
            }
        });
    </script>
```

If you inspect the element for the modal, you see that the `display: none` is set for the _iframe_ generated for that modal.

Is this a bug or is there a problem with that when testing in localhost?

Bellow follows the image.

![cookie_modal](https://user-images.githubusercontent.com/5311651/201192291-16e5360c-e7b6-4318-b44c-90c3e749aa57.png)

**Comment by icaldana** [comment]
Hi @caco13! 😺

We apologize for the inconvenience in your workflow.
Even though this space is open, it is not intended to be a tool to support unexpected errors of the platform and its integrations.

We have an official channel prepared to solve your case quickly.

So if the error still persists, please pile up all the information and access the following link to share the problem with us: https://www.mercadopago.com.br/developers/pt/support/center/tickets

Thank you so much for your contact!
